atest/testdata/test_libraries/dynamic_libraries/EmbeddedArgs.py

```diff
@@ -0,0 +1,26 @@
+from robot.api import deco
+from robot.errors import DataError
+
+
+class EmbeddedArgs(object):
+
+    def callables(self):
+        return [getattr(self, f) for f in dir(self) if callable(getattr(self, f))]
+
+    def get_keyword_names(self):
+        return [f.robot_name for f in self.callables() if hasattr(f, 'robot_name')]
```

Comment:
This is pretty complex. Couldn't this method just return `['Add ${count} Copies Of ${item} To Cart']`?